src/Tribe/Tickets_Handler.php

```diff
@@ -559,7 +566,7 @@ public function attendees_admin_body_class( $body_classes ) {
 	 *
 	 * @return string
 	 */
-	public function attendees_admin_title( $admin_title, $title ) {
+	public function attendees_admin_title( $admin_title, $unused_title ) {
```

Comment:
would encourage changing the `@param $title` section to be `@param $unused_title` to match the change in param name here